scapy/contrib/isotp.py

```diff
@@ -510,21 +578,25 @@ def _feed_consecutive_frame(self, identifier, ea, data):
         return True
 
     def _feed_flow_control_frame(self, identifier, ea, data):
+        # type: (int, Optional[int], bytes) -> bool
         if len(data) < 3:
             # At least 2 bytes are necessary: 1 for sequence number and
             # 1 for data
             return False
 
-        keys = [self.last_ff, self.last_ff_ex]
-        if not any(keys):
-            return False
+        keys = list()  # type: List[Tuple[Optional[int], int, int]]
+
+        if self.last_ff is not None:
+            keys.append(self.last_ff)
+        if self.last_ff_ex is not None:
+            keys.append(self.last_ff_ex)
```

Comment:
`keys = [bool(x) for x in (self.last_ff, self.last_ff_ex)]`
might be worth a shot as a one-liner

Reply:
thanks